etc/profile-a-l/evolution.profile

```diff
@@ -40,7 +72,25 @@ novideo
 protocol unix,inet,inet6
 seccomp
 shell none
+tracelog
 
+disable-mnt
+# Add "gpg,gpg2,gpg-agent,pinentry-curses,pinentry-emacs,pinentry-fltk,pinentry-gnome3,pinentry-gtk,pinentry-gtk2,pinentry-gtk-2,pinentry-qt,pinentry-qt4,pinentry-tty,pinentry-x2go,pinentry-kwallet" for gpg
+private-bin evolution
```

Comment:
Probably breaks others things too?

Reply:
I haven't seen except browsers,I used plain-text emails. I'll comment bin and remove the gpg line?

Reply:
I would say add the gpg stuff to the commented private-bin for references if someone wants to use private-bin.